backends/advanced/src/advanced_omi_backend/controllers/system_controller.py

```
@@ -455,3 +455,103 @@ async def set_memory_provider(provider: str):
+        if not isinstance(prompt_text, str):
+            return {"valid": False, "error": "Prompt must be a string"}
+
+        prompt_text = prompt_text.strip()
+        if len(prompt_text) < 10:
+            return {"valid": False, "error": "Prompt too short (minimum 10 characters)"}
+        if len(prompt_text) > 10000:
+            return {"valid": False, "error": "Prompt too long (maximum 10000 characters)"}
+
+        return {"valid": True, "message": "Configuration is valid"}
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)